types/adm-zip/index.d.ts

```diff
@@ -65,14 +65,14 @@ declare class AdmZip {
      * @param callback Called with the resulting string.
      * @param encoding Optional. If no encoding is specified utf8 is used
      */
-    readAsTextAsync(fileName: string, callback: (data: string) => any, encoding?: string): void;
+    readAsTextAsync(fileName: string, callback: (data: string, err?: any) => any, encoding?: string): void;
```

Comment:
type signature for second callback argument is `err?: string`, same below
ref: https://github.com/cthackers/adm-zip/blob/576f7c5a5ccd529c75c0c7cf870a69856d99ca3a/util/errors.js#L9

Reply:
Actually, unless I'm missing something, it doesn't even look like adm-zip calls the callback with an error.

https://github.com/cthackers/adm-zip/blob/3f00a03ff55a1e1643b690a67fbe164e9ed7f48a/adm-zip.js#L113